modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,496 @@
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_db_create_setVectorProperty[`db.create.setVectorProperty()`] label:beta[]
+| {check-mark}
+| {check-mark}
+| label:deprecated[Deprecated in 5.13] +
+Replaced by: xref:procedures.adoc#procedure_db_create_setNodeVectorProperty[`db.create.setNodeVectorProperty()`]
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_upgrade[`dbms.upgrade()`] label:admin-only[]
+| {check-mark}
+| {check-mark}
+| label:deprecated[Deprecated in 5.9]
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_upgradestatus[`dbms.upgradeStatus()`] label:admin-only[]
+| {check-mark}
+| {check-mark}
+| label:deprecated[Deprecated in 5.9]
+
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_cluster_readreplicatoggle[`dbms.cluster.readReplicaToggle()`] label:admin-only[]
+|
+| {check-mark}
+| label:deprecated[Deprecated in 5.6] +
+Replaced by: xref:procedures.adoc#procedure_dbms_cluster_secondaryreplicationdisable[`dbms.cluster.secondaryReplicationDisable()`].
+|===
+====
+
+=== Configuration settings
+
+.See all deprecated configuration settings
+[%collapsible]
+====
+[options=header, cols="3m,1,1,2"]
+|===
+| Name
+| Community Edition
+| Enterprise Edition
+| Comment
+
+| xref:configuration/configuration-settings.adoc#config_server.db.query_cache_size[`server.db.query_cache_size`]
+| {check-mark}
+| {check-mark}
+| label:deprecated[Deprecated in 5.7]
+
+| xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.auth_params[`dbms.security.oidc.<provider>.auth_params`]
+|
+| {check-mark}
+| label:dynamic[] label:deprecated[]
+
+| xref:configuration/configuration-settings.adoc#config_dbms.security.oidc.-provider-.client_id[`dbms.security.oidc.<provider>.client_id`]
+|
+| {check-mark}
+| label:dynamic[] label:deprecated[]
+|===
+====
+
+[role=label--enterprise]
+=== Metrics
+
+.See all deprecated metrics
+[%collapsible]
+====
+[options=header, cols="3m,3"]
+|===
+| Name
+| Comment
+
+| xref:monitoring/metrics/reference.adoc#db-data-metrics[Database data metrics]
+| label:deprecated[Deprecated in 5.15]
+|===
+====
+
+=== Neo4j-admin tool
+
+.See all deprecated commands
+[%collapsible]
+====
+[options=header, cols="3m,1,1,3"]
+|===
+| Name
+| Community Edition
+| Enterprise Edition
+| Comment
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/backup-restore/aggregate[`neo4-admin database aggregate-backup`]
+|
+| {check-mark}
+| label:deprecated[Deprecated in 2025.01] +
+Replaced by xref:backup-restore/aggregate.adoc[`neo4j-admin backup aggregate`]
+|===
+====
+
+=== Seed providers
+
+`S3SeedProvider` is deprecated since Neo4j 5.26.
+For more information, refer to the xref:clustering/databases.adoc#s3-seed-provider[Seed from URI] documentation on the page xref:clustering/databases.adoc[Managing databases in a cluster].
+
+== Breaking changes
```

Comment:
@renetapopova, do we need to say something about logging here?

Reply:
Yeah, that's a good point.
You can add something like:
```
Starting from Neo4j 2025.01, the default _debug.log_ format has been changed from Text to JSON.
It is highly recommended to keep it enabled and not to alter it. +
If you require the _debug.log_ messages in a different format, create an additional Appender since the Neo4j Support team uses the default one.
For more information, see xref:monitoring/logging.adoc##_default_logging_configuration[Default logging configuration].
```